### PR TITLE
PIM-6333: Create product model standard normalizer

### PR DESCRIPTION
**Pull request by aRn0D**
**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | yes
| Added Behats                      | -
| Added integration tests           | -
| Changelog updated                 | -
| Review and 2 GTM                  | -
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

### CODE REVIEW

File: src/Pim/Component/Catalog/Normalizer/Standard/ProductModelNormalizer.php [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<?php
+declare(strict_types=1);
```
Comment (damien-carcel) [inline src/Pim/Component/Catalog/Normalizer/Standard/ProductModelNormalizer.php]: ❤️

File: src/Pim/Component/Catalog/Normalizer/Standard/ProductModelNormalizer.php [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+namespace Pim\Component\Catalog\Normalizer\Standard;
+
+use Pim\Component\Catalog\Model\ProductModelInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+use Symfony\Component\Serializer\Normalizer\SerializerAwareNormalizer;
+
+/**
+ *
```
Comment (damien-carcel) [inline src/Pim/Component/Catalog/Normalizer/Standard/ProductModelNormalizer.php]: To remove

File: src/Pim/Component/Catalog/Normalizer/Standard/ProductModelNormalizer.php (hunk 1 above)
Comment (jjanvier) [inline src/Pim/Component/Catalog/Normalizer/Standard/ProductModelNormalizer.php]: not sure about that ^^

Comment (aRn0D) [inline src/Pim/Component/Catalog/Normalizer/Standard/ProductModelNormalizer.php]: why ?

**Pull request merged**